test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatLocalsTests.cs

```diff
@@ -100,19 +101,29 @@ public static void Locals_Clear_Succeeds(string args)
                     {
                         { "NUGET_PACKAGES", mockGlobalPackagesDirectory.FullName },
                         { "NUGET_HTTP_CACHE_PATH", mockHttpCacheDirectory.FullName },
-                        { RuntimeEnvironmentHelper.IsWindows ? "TMP" : "TMPDIR", mockTmpCacheDirectory.FullName }
+                        { RuntimeEnvironmentHelper.IsWindows ? "TMP" : "TMPDIR", mockTmpDirectory.FullName }
                     });
                 // Unix uses TMPDIR as environment variable as opposed to TMP on windows
 
                 // Assert
                 if (StringComparer.Ordinal.Equals(cacheType, "all"))
                 {
                     Assert.False(Directory.Exists(mockGlobalPackagesDirectory.FullName));
-                    var x = Directory.EnumerateFileSystemEntries(mockGlobalPackagesDirectory.FullName);
-                    //Assert.Equal(Directory.EnumerateFileSystemEntries(mockGlobalPackagesDirectory.FullName), IEnumerable<String>);
                     Assert.False(Directory.Exists(mockHttpCacheDirectory.FullName));
                     Assert.False(Directory.Exists(mockTmpCacheDirectory.FullName));
                 }
+                else if (StringComparer.Ordinal.Equals(cacheType, "global-packages"))
+                {
+                    Assert.False(Directory.Exists(mockGlobalPackagesDirectory.FullName));
```

Comment:
Assert that the test files created above `DotnetCliUtil.CreateTestFiles` were not deleted. Basically, in this particular case, the `temp` and `http-cache` directories should have the same file system entires as before running the command.